dags/copy_deduplicate.py

```diff
@@ -87,49 +87,7 @@
 
     copy_deduplicate_main_ping >> bq_main_events
 
-    # Experiment enrollment aggregates chain (depends on events)
-
-    experiment_enrollment_aggregates = bigquery_etl_query(
-        task_id="experiment_enrollment_aggregates",
-        project_id="moz-fx-data-shared-prod",
-        destination_table="experiment_enrollment_aggregates_v1",
-        dataset_id="telemetry_derived",
-        owner="ssuh@mozilla.com",
-        email=["telemetry-alerts@mozilla.com", "ssuh@mozilla.com"])
-
-    gen_query_task_id = "experiment_enrollment_aggregates_live_generate_query"
-
-    # setting xcom_push to True outputs this query to an xcom
-    experiment_enrollment_aggregates_live_generate_query = gke_command(
-        task_id=gen_query_task_id,
-        command=[
-            "python",
-            "sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_live/view.sql.py",
-            "--submission-date",
-            "{{ ds }}",
-            "--json-output",
-            "--wait-seconds",
-            "15",
-        ],
-        docker_image="mozilla/bigquery-etl:latest",
-        xcom_push=True,
-        owner="ssuh@mozilla.com",
-        email=["telemetry-alerts@mozilla.com", "ssuh@mozilla.com"])
-
-    experiment_enrollment_aggregates_live_run_query = bigquery_xcom_query(
-        task_id="experiment_enrollment_aggregates_live_run_query",
-        destination_table=None,
-        dataset_id="telemetry_derived",
-        xcom_task_id=gen_query_task_id,
-        owner="ssuh@mozilla.com",
-        email=["telemetry-alerts@mozilla.com", "ssuh@mozilla.com"])
-
-    bq_main_events >> experiment_enrollment_aggregates
-    (event_events >>
-     experiment_enrollment_aggregates >>
-     experiment_enrollment_aggregates_live_generate_query >>
-     experiment_enrollment_aggregates_live_run_query)
-
+    # todo: remove
```

Comment:
Experiment search aggregate data will be exported as part of the experiment monitoring data export DAG as soon as the queries have been migrated (in a future PR).